### ISSUE TITLE
virtualbox/eval-win7x64-enterprise-cygwin broken

### ISSUE THREAD

**Issue by destijl** [boxcutter/windows#10]
Hi, I'm using virtualbox 4.3.26 (i.e. the latest), and the latest repo version.  I can make "virtualbox/eval-win7x64-enterprise" fine, but the cygwin version of the same thing fails.  I poked around in the scripts but it isn't obvious why this is failing at this point.

$ make virtualbox/eval-win7x64-enterprise-cygwin
rm -rf output-virtualbox-iso
mkdir -p box/virtualbox
packer build -only=virtualbox-iso -var 'cm=nocm' -var 'version=1.0.4' -var 'update=true' -var 'headless=false' -var "shutdown_command=shutdown /s /t 10 /f /d p:4:1 /c 'Packer Shutdown'" -var "iso_url=http://care.dlservice.microsoft.com/dl/download/evalx/win7/x64/EN/7600.16385.090713-1255_x64fre_enterprise_en-us_EVAL_Eval_Enterprise-GRMCENXEVAL_EN_DVD.iso" -var "iso_checksum=15ddabafa72071a06d5213b486a02d5b55cb7070" eval-win7x64-enterprise-cygwin.json
virtualbox-iso output will be in this color.

==> virtualbox-iso: Downloading or copying Guest additions
    virtualbox-iso: Downloading or copying: file:///usr/share/virtualbox/VBoxGuestAdditions.iso
==> virtualbox-iso: Downloading or copying ISO
    virtualbox-iso: Downloading or copying: http://care.dlservice.microsoft.com/dl/download/evalx/win7/x64/EN/7600.16385.090713-1255_x64fre_enterprise_en-us_EVAL_Eval_Enterprise-GRMCENXEVAL_EN_DVD.iso
==> virtualbox-iso: Creating floppy disk...
    virtualbox-iso: Copying: floppy/win7x64-enterprise/Autounattend.xml
    virtualbox-iso: Copying: floppy/00-run-all-scripts.cmd
    virtualbox-iso: Copying: floppy/install-winrm.cmd
    virtualbox-iso: Copying: floppy/fixnetwork.ps1
    virtualbox-iso: Copying: floppy/powerconfig.bat
    virtualbox-iso: Copying: floppy/01-install-wget.cmd
    virtualbox-iso: Copying: floppy/_download.cmd
    virtualbox-iso: Copying: floppy/_packer_config.cmd
    virtualbox-iso: Copying: floppy/passwordchange.bat
    virtualbox-iso: Copying: floppy/networkprompt.bat
    virtualbox-iso: Copying: floppy/disablewinupdate.bat
    virtualbox-iso: Copying: floppy/cygwin.bat
    virtualbox-iso: Copying: floppy/cygwin.sh
    virtualbox-iso: Copying: floppy/zz-start-sshd.cmd
    virtualbox-iso: Copying: floppy/oracle-cert.cer
==> virtualbox-iso: Creating virtual machine...
==> virtualbox-iso: Creating hard drive...
==> virtualbox-iso: Attaching floppy disk...
==> virtualbox-iso: Creating forwarded port mapping for SSH (host port 3484)
==> virtualbox-iso: Executing custom VBoxManage commands...
    virtualbox-iso: Executing: modifyvm eval-win7x64-enterprise-cygwin --memory 2048
    virtualbox-iso: Executing: modifyvm eval-win7x64-enterprise-cygwin --cpus 1
==> virtualbox-iso: Starting the virtual machine...
==> virtualbox-iso: Waiting 10s for boot...
==> virtualbox-iso: Typing the boot command...
==> virtualbox-iso: Waiting for SSH to become available...
==> virtualbox-iso: Connected to SSH!
==> virtualbox-iso: Uploading VirtualBox version info (4.3.26)
==> virtualbox-iso: Uploading VirtualBox guest additions ISO...
==> virtualbox-iso: Provisioning with shell script: script/vagrant.bat
    virtualbox-iso: 'C:/Windows/Temp/script.bat' is not recognized as an internal or external command,
    virtualbox-iso: operable program or batch file.
==> virtualbox-iso: Unregistering and deleting virtual machine...
==> virtualbox-iso: Deleting output directory...
Build 'virtualbox-iso' errored: Script exited with non-zero exit status: 1

**Comment by rickard-von-essen** [comment]
I reproduced this.

I fixed it by applying:
```
diff --git a/eval-win81x64-enterprise-cygwin.json b/eval-win81x64-enterprise-cygwin.json
index 6263996..ae52cb1 100644
--- a/eval-win81x64-enterprise-cygwin.json
+++ b/eval-win81x64-enterprise-cygwin.json
@@ -146,7 +145,7 @@
         "UPDATE={{user `update`}}"
       ],
       "execute_command": "{{.Vars}} cmd /c C:/Windows/Temp/script.bat",
-      "remote_path": "/tmp/script.bat",
+      "remote_path": "C:/Windows/Temp/script.bat",
       "scripts": [
         "script/vagrant.bat",
         "script/cmtool.bat",
@@ -157,12 +156,6 @@
         "script/sdelete.bat"
       ],
       "type": "shell"
-    },
-    {
-      "inline": [
-        "rm -f /tmp/script.bat"
-      ],
-      "type": "shell"
     }
   ],
   "variables": {
```

The inline should probably be kept and fixed, but I'm not sure if it will run under cmd or cygwin.

**Comment by TheBigBear** [comment]
I tested the following PR and it fixes this one and #35 see https://github.com/boxcutter/windows/pull/44

**Comment by arizvisa** [comment]
Closing this as it appears this issue was fixed just not closed by the OP.

If you believe this to be a mistake, lmk.
